src/tests/ftest/container/fill_destroy_loop.py

```diff
@@ -79,35 +144,50 @@ def test_fill_destroy_cont_loop(self):
             any other error.
 
         Use Case:
-            repeat following steps:
-            (1)Create Pool and Container.
-            (2)Fill the pool with random block data size, verify return code is as expected
-               when no more data can be written to the container.
-            (3)Pool free space before writing data to container
-            (4)Check for DER_NOSPACE -1007.
-            (5)Display bytes written when pool is full before container delete.
-            (6)Display free space before container delete.
-            (7)Display and verify free space after container delete.
+            - Create Pool
+            - repeat following steps:
+              (0) Set reclaim pool property
+              (1) Create Container.
+              (2) Fill the pool with random block data size, verify return code is as expected
+                  when no more data can be written to the container.
+              (3) Pool free space before writing data to container
```

Comment:
I don't understand this text, should it be "Measure pool space ..." or "Free pool space before writing more data" ? Or maybe this is about displaying and should go before 2?

Reply:
Oups s/Pool/Poll/

- [ ] s/Pool/Measure/